src/canvas/Canvas.ts

```diff
@@ -114,7 +114,7 @@ export class Canvas extends SelectableCanvas {
 
   declare currentSubTargets?: FabricObject[];
 
-  private _isClick: boolean;
+  private _isClick = false;
```

Comment:
this line here actually is:
```
    _defineProperty(this, "_isClick", void 0);
```
with your change becomes `false` instead of void 0.

If the issue for you is this being undefine, just add `this._isClcik = false` in the constructor.

If that is not an issue what we want here is a declare.

I'm also not sure if the correct default value is false or true, since we seem to make at first event it a  `true` unless you change it to a `false` with a drag or a move.